mmx-tests/Makefile

```diff
@@ -0,0 +1,29 @@
+USE_XMM=
+#USE_XMM=--use-xmm
```

Comment:
There are compilation errors when I try this option:
```
test.c:187:34: error: passing '__m64' (vector of 1 'long long' value) to parameter of incompatible type '__m128' (vector of 4 'float' values)
  187 |     orig__mm_stream_pi(&mem.m64, (__m64)mmx_vals[l0]);
      |                                  ^~~~~~~~~~~~~~~~~~~
./gen_orig.h:83:47: note: passing argument to parameter 'arg_1' here
   83 | void orig__mm_stream_pi(__m64 * arg_0, __m128 arg_1);
      |                                               ^
test.c:190:34: error: passing '__m64' (vector of 1 'long long' value) to parameter of incompatible type '__m128' (vector of 4 'float' values)
  190 |     new__mm_stream_pi(&mem2.m64, (__m64)mmx_vals[l0]);
      |                                  ^~~~~~~~~~~~~~~~~~~
./gen_new.h:83:46: note: passing argument to parameter 'arg_1' here
   83 | void new__mm_stream_pi(__m64 * arg_0, __m128 arg_1);
      |                                              ^
test.c:213:27: error: passing '__m64' (vector of 1 'long long' value) to parameter of incompatible type '__m128' (vector of 4 'float' values)
  213 |     new__mm_maskmove_si64((__m64)0x1122334455667788LL, (__m64)0x8000808000800000, copy_location);
      |                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~
./gen_new.h:90:35: note: passing argument to parameter 'arg_0' here
   90 | void new__mm_maskmove_si64(__m128 arg_0, __m128 arg_1, char * arg_2);
      |                                   ^
3 errors generated.
make: *** [Makefile:24: test.o] Error 1
```

Reply:
It works if you first delete/ifdef-out test_stores and test_maskmove. I didn't make those generic.

(The "USE_XMM" version is to validate that the unused bits of the input xmm registers being set to arbitrary values won't cause trouble, since some of the instructions trigger side-effects, like floating-point exception flags being set.)